StrongInject.Generator/Extensions.cs

```diff
@@ -35,6 +35,22 @@ public static IEnumerable<(T value, int index)> WithIndex<T>(this ImmutableArray
             }
             return value;
         }
+        
+        public static bool TryAdd<TKey, TValue>(this Dictionary<TKey, TValue> dic, TKey key, TValue value, out TValue existingValue)
```

Comment:
I would have expected a compiler warning to require this to be `[MaybeNullWhen(false)] out TValue existingValue`.

Reply:
I'm on .Net Standard 2.0, and just never got round to adding the attributes. For now I just supress the warnings when they come up, but will get round to doing this properly at some point.